src/main/java/com/example/demo/DemoApplication.java

```diff
@@ -36,6 +36,7 @@ public String getWelcomebye() {
 	}
 	@RequestMapping("/byej")
 	public String getWelcomebyeh() {
+		System.out.println("rahul's commit");
```

Comment:
move this hardcoded value to constant file

Reply:
Done